From a04d730bb7ba63df17117c3bfc4afd93bab9f37c Mon Sep 17 00:00:00 2001
From: James McCoy <jamessan@debian.org>
Date: Fri, 24 Feb 2017 23:43:57 -0500
Subject: [PATCH] excuse: Add an EOL to the verdict summary line in HTML output

devscripts' grep-excuses expects each <li> to be on its own line.  When
d7a676d0741729bb643e0b8c54b989cb747c6a4b added the verdict summary,
without an EOL, it broke grep-excuses' ability to search by maintainer.

Signed-off-by: James McCoy <jamessan@debian.org>
---
 britney2/excuse.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/britney2/excuse.py b/britney2/excuse.py
index 4dbd703..e301cfe 100644
--- a/britney2/excuse.py
+++ b/britney2/excuse.py
@@ -182,7 +182,7 @@ class Excuse(object):
         """Render the excuse in HTML"""
         res = "<a id=\"%s\" name=\"%s\">%s</a> (%s to %s)\n<ul>\n" % \
             (self.name, self.name, self.name, self.ver[0], self.ver[1])
-        res += "<li>Migration status: %s" % self._format_verdict_summary()
+        res += "<li>Migration status: %s\n" % self._format_verdict_summary()
         if self.maint:
             res = res + "<li>Maintainer: %s\n" % (self.maint)
         if self.section and self.section.find("/") > -1:
-- 
2.11.0

